webapp/components/post_view/post_focus_view_controller.jsx

```diff
@@ -103,7 +107,15 @@ export default class PostFocusView extends React.Component {
     }
 
     onPreferenceChange() {
+        const focusedPostId = PostStore.getFocusedPostId();
+        if (focusedPostId == null) {
+            return;
```

Comment:
As far as I can tell it will always have a focused post unless im missing something, but actually this controller is loaded for permalinks, couldn't find somewhere else that loads this controller